test/resources/ostree.resource

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Documentation    Keywords for OSTree-based systems
+
+Library     SSHLibrary
+
+
+*** Variables ***
+${BACKUP_STORAGE}       /var/lib/microshift-backups
+${HEALTH_FILE}          /var/lib/microshift-backups/health.json
+
+
+*** Keywords ***
+Get Booted Deployment Status
+    [Documentation]    Get rpm-ostree status of booted deployment as a json
+
+    ${ostree_status}    ${rc}=    Execute Command
+    ...    rpm-ostree status --booted --json
+    ...    return_rc=True
+    Should Be Equal As Integers    0    ${rc}
+    ${json}=    Evaluate    json.loads('''${ostree_status}''')    json
+    RETURN    ${json}
+
+Get Booted Deployment ID
+    [Documentation]    Get ID of currently booted deployment
+
+    ${status}=    Get Booted Deployment Status
+    RETURN    ${status["deployments"][0]["id"]}
```

Comment:
This uses kw `Get Booted Deployment Status` which runs `rpm-ostree status --booted --json`
Thanks to `--booted` it will only output one deployment, but it's still using the same data structure, i.e. `deployments` in an array. With `--booted` it will only have 1 element.

Without that switch, `0` could be either booted or staged deployment